vendor/github.com/mendersoftware/progressbar/progress.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package progressbar
+
+// TODO -- Add terminal width respect
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/mattn/go-isatty"
+	// "golang.org/x/sys/unix" - Do not add for now (split into minimal and not (?))
```

Comment:
?

Reply:
I left this in, as the package is still in pre-stable (ie, currently on `v.0.0.1`.). This will be needed for the next thing I want in, before `1.0`, and that is automatic detection of `TTY` size. This will be cleaned up.